Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM alpine:3.10
+LABEL maintainer "Wes Lambert, wlambertts@gmail.com"
+LABEL version="CyberChef server 0.0.1"
+LABEL description="Dockerized version of Cyberchef server (from the fine folks at GCHQ: https://github.com/gchq/CyberChef-server)"
+RUN apk update && apk add --no-cache nodejs nodejs-npm && \
```

Comment:
If you ran from a node-alpine image (preferrably node 10), we could be more explicit about which version of node we install (we have not been able to confirm full functionality with anything. later than node 10).

Reply:
This would also solve the problem where there is no `nodejs-npm` package for this version of alpine: https://superuser.com/questions/1125969/how-to-install-npm-in-alpine-linux

Reply:
Thanks, I will take a look at this!

Reply:
Updated in PR.